dev/doc/build-system.dune.md

```diff
@@ -175,6 +175,15 @@ local copy of Coq. For this purpose, Dune supports the `-p` option, so
 version of Coq libs, and use a "release" profile that for example
 enables stronger compiler optimizations.
 
+## Native compute and native compilation
+
+Dune provides experimental support for native compilation, enabled
+using the `(mode native)` field in the `(coq.theory )` stanza. Given
+that native compilation is quite expensive, Dune won't compile native
+objects if the `dev` profile is the active one.
```

Comment:
Is this a general feature of Dune native compilation support?

Reply:
What do you mean by general @Zimmi48 ? For now the rule governing this hardcoded but actually it would be pretty easy to do add a per-profile setting.

Reply:
I wanted to know if this was hard-coded in Dune because I couldn't see what could be triggering this in the code of Coq.

Reply:
Yes this is hardcoded in Dune, when in `dev` profile.